### DISCUSSION TITLE
Any plan to release 7.x ?

### DISCUSSION THREAD

**Discussion by carlittoer** [deployphp/deployer#3099] [General]
Hi, I want to give deployer a shot. It looks very useful to my setup. 

7.x is still not released and the doc is not fully complete?

Any plans to release 7.x soon ?

Thanks !

**Comment by antonmedv** [comment]
Yes! Release is really soon.

**Reply by benoitchantre** [reply]
Is there something specific that blocks the release?

**Reply by antonmedv** [reply]
Nope) just need to find time for it.

**Comment by andyexeter** [comment]
A stable release would be awesome :)

Currently, [deployer.org](https://deployer.org/) defaults to v7.x documentation - but when you follow the v7.x installation guide by running:

```
composer require --dev deployer/deployer
```
You get v6.8 because of composer's minimum stability defaulting to `stable`. However, when changing deployer.org to 6.x it states that this version is no longer actively maintained. I think it is confusing for newcomers.

The only way to get v7 installed is by first adding `"minimum-stability": "RC"` to the project's composer.json file and then running the above `require` command.

**Comment by carlittoer** [comment]
Hello again !

We are waiting eagerly for the new release. Is it coming any time soon ?

Thanks !

**Reply by antonmedv** [reply]
A hope too) 

Let’s do this: if I got 30 GitHub Sponsors, I will release v7 immediately))

**Comment by carlittoer** [comment]
You mean sponsors ?

**Reply by antonmedv** [reply]
Yes))

**Comment by Metis77** [comment]
Sponsor? Check.
I Love deployer ... and hopefully even more will support it.

**Comment by carlittoer** [comment]
Check too!
